ui/apps/dashboard/src/components/Navigation/Environments.tsx

```diff
@@ -110,10 +109,15 @@ export default function EnvironmentSelectMenu({
   const router = useRouter();
   const [selected, setSelected] = useState<Environment | null>(null);
   const nextPathname = useSwitchablePathname();
-  const [{ fetching, data: envs = [] }] = useEnvironments();
-
-  if (fetching || !isNonEmptyArray(envs)) {
-    return <Skeleton className={`h-8 ${collapsed ? 'w-8 px-1' : 'w-[146px] px-2'}`} />;
+  const [{ data: envs = [], error }] = useEnvironments();
+
+  if (error) {
+    console.error('error fetching envs', error);
+    return (
+      <Alert severity="error" className="mx-auto flex h-8 w-full items-center text-sm">
+        Env Error
```

Comment:
```suggestion
        Error loading environments. Please try again or contact support if the issue does not resolve.
```

Reply:
I'll move it to a tooltip since that is a very space constrained widget.